### PR TITLE
highlight new keywords

### PR DESCRIPTION
**Pull request by jeffersoncarpenter**
For Issue #418

### CODE REVIEW

**Comment by jeffersoncarpenter** [comment]
That build failure doesn't look related to the patch; it fails while cabal installing Idris.

File: idris-syntax.el [hunk 1]
```diff
@@ -100,7 +100,7 @@ contributing the settings upstream to the theme maintainer."
 
 
 (defvar idris-definition-keywords
-  '("data" "class" "codata" "constructor" "record" "postulate")
+  '("data" "class" "codata" "constructor" "interface" "record" "postulate")
```
Comment (david-christiansen) [inline idris-syntax.el]: Shouldn't `class` be removed from this list now?

File: idris-syntax.el [hunk 2]
```diff
@@ -144,8 +144,8 @@ contributing the settings upstream to the theme maintainer."
     st))
 
 (defconst idris-keywords
-  '("abstract" "case" "covering" "default" "do" "dsl" "else" "if"
-    "implicit" "import" "in" "infix" "infixl" "infixr" "instance"
+  '("abstract" "case" "covering" "default" "do" "dsl" "else" "export" "if"
+    "implementation" "implicit" "import" "in" "infix" "infixl" "infixr" "instance"
```
Comment (david-christiansen) [inline idris-syntax.el]: Shouldn't `instance` be removed?

**Comment by jeffersoncarpenter** [comment]
Yeah, `class` and `instance` should be removed.  I had left them in, so as not to break highlighting for old code.

**Comment by david-christiansen** [comment]
All right. Generally I try to keep `idris-mode` corresponding to the latest Idris release, and if people have old code, they should use an old version of the mode with an old Idris.

In any case, I appreciate the update and I'll take a look at the CI stuff when I get some time.

**Pull request merged**